src/test/kotlin/com/terraformation/backend/report/ReportServiceTest.kt

```diff
@@ -337,6 +337,14 @@ class ReportServiceTest : DatabaseTest(), RunsAsUser {
               remainingUnitsId = SeedQuantityUnits.Seeds,
           ),
       )
+      insertAccession(
+          AccessionsRow(
+              facilityId = nonProjectSeedBankId,
```

Comment:
should this be `otherProjectSeedbankId`? I see L#334 as well in the older code.. maybe this is fine.

Reply:
Since seed banks aren't directly tied to projects, I am pretty sure this is functionally the same

Reply:
But I'll wait for @sgrimm to confirm since I'm not certain

Reply:
Test is certainly functionally correct.
Comment was mostly from parsing the const name, assuming it was earmarked for testing without projects. All good! Nothing to follow up.